terminalT.html

```diff
@@ -1,261 +1,350 @@
+            } else if (Key == 69) { //e
+                letterCheck("e", "E");
+            } else if (Key == 48) { //0
+                otherCheck("0", ")");
+            } else if (Key == 49) { //1
+                otherCheck("1", "!");
+            } else if (Key == 50) { //2
+                otherCheck("2", "@");
+            } else if (Key == 51) {
+                otherCheck('3', "#");
+            } else if (Key == 52) {
+                otherCheck('4', "$");
+            } else if (Key == 53) {
+                otherCheck('5', "%");
+            } else if (Key == 54) {
+                otherCheck("6", "^");
+            } else if (Key == 55) {
+                otherCheck('7', "&");
+            } else if (Key == 56) {
+                otherCheck("8", "*");
+            } else if (Key == 57) { //9
+                otherCheck("9", "(");
+            } else if (Key == 8) {
+                backSpace()
+            }
+            /*else if(Key==58){
+
+            			}*/
+        }
+        //END old typeOnDisplay
+
+        /* for shift character change
+        				//for letters:
+        				if(sP==1&&cL==1){
+        					addToDisplay("_");//lower
+        				} else if(sP==1||cL==1){
+        					addToDisplay("_");//upper
+        				} else {
+        					addToDisplay("_");//lower
+        				}
+        				//for the not letters:
+        				if(sP==1){
+        					addToDisplay("_");//alt
+        				} else {
+        					addToDisplay("_");//normal
+        				}
+
+        */
+
+
+        //START replacement backSpace
+        function backSpace() {
+            let text = Display.innerHTML;
+            console.log(text);
+            if (text) {
+                Display.innerHTML = text.substring(0, text.length - 1);
+                console.log(text);
+            }
+        }
+        //END replacement backSpace
+
+        //START old backSpace
+        function oldBackSpace() {
+            let t = Display.innerHTML;
+            t = t.split("");
+            let t2 = "";
+            for (i = 0; i < t.length - 1; i++) {
+                t2 = t2 + t[i];
+                console.log("T letter:" + t[i]);
+                console.log("full string:" + t2);
+            }
+            if (Array.isArray(t2)) {
+                Display.innerHTML = "fatal removal error, please restart the terminal."
+                return;
+            } else {
+                Display.innerHTML = t2;
+            }
+        }
+        //END old backSpace
+
+        function letterCheck(lower, upper) { //l is the lower case character, U is the uppercase character
+            if ((sP == 1) != (cL == 1)) {
+                addToDisplay(upper);
+            } else {
+                addToDisplay(lower);
+            }
+        }
+
+        function otherCheck(N, S) { //N is for the normal character, S is for the special character.
+            if (sP == 1) {
+                Display.innerHTML = Display.innerHTML + S;
```

Comment:
changed this (and similar lines) to call "addToDisplay", mind changing that to match?